### ISSUE TITLE
Instructions for Alexa skill deployment stale and many points do not work

### ISSUE THREAD

**Issue by haji-haji-haji** [Amir974/echo-sensibo-skill#1]
Hi @Amir974 

I know its been 3 years since you have worked on this project, but please if you can:

Possible if you can log into your Alexa Dexelopment console and export the JSON please?

You can no longer upload files as per before since they changed from V2 to V3.

Same time I can then modify invocations etcetera to suit Japan as Sensibo have released a product in Japan but they refuse to update their skill so it can be on the Japanese Alexa skill set but also update instructions too.

Kind regards,

Haji

**Comment by Amir974** [comment]
Not sure what I can do about it to be honest...
I haven’t been using this skill myself since sensibo created their own official skill.
I guess i should just pull out this whole thing from github

**Comment by haji-haji-haji** [comment]
Yes, but they haven't made a skill for Japan and I think they are too lazy. ~ I even offered to do translations for free.

Kind regards,

Haji

**Comment by Amir974** [comment]
Well I reached out to Sensibo CEO who I am in contact with
I understand it's a problem since Amazon Japan does not yet support "Thermostat" type devices.

anyway, feel free to fork and fix my code if it helps you!

**Comment by haji-haji-haji** [comment]
Hi @Amir974,

Actually, they have other devices already online like Daikin and a few others. It might not come under thermostat but it should be okay for them to add now to the skills marketplace.

Yes, I am going to fork and update where needed :) I will also include the JSON file too so it simplifies the process.

@haji-haji-haji
